2018-05-31 09:45:33,572 ERROR: Exception on /user/sarah [GET] [in c:\users\william\documents\microblog\venv\lib\site-packages\flask\app.py:1761
Traceback (most recent call last):
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\William\Documents\microblog\app\routes.py", line 79, in user
    return render_template('user.html', user=user, posts=posts.items, next_url=next_url, prev_url=prev_url)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\flask\templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\William\Documents\microblog\app\templates\user.html", line 47, in template
    <p>Last seen on: {{ moment(user.last_seen).format('LLL')</p>
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 213, in parse_if
    'name:endif'))
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 875, in subparse
    add_data(self.parse_tuple(with_condexpr=True))
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 620, in parse_tuple
    args.append(parse())
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 432, in parse_expression
    return self.parse_condexpr()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 437, in parse_condexpr
    expr1 = self.parse_or()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 450, in parse_or
    left = self.parse_and()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 459, in parse_and
    left = self.parse_not()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 470, in parse_not
    return self.parse_compare()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 480, in parse_compare
    ops.append(nodes.Operand(token_type, self.parse_math1()))
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 496, in parse_math1
    left = self.parse_concat()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 507, in parse_concat
    args = [self.parse_math2()]
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 517, in parse_math2
    left = self.parse_pow()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 528, in parse_pow
    left = self.parse_unary()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 546, in parse_unary
    node = self.parse_primary()
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 583, in parse_primary
    self.fail("unexpected '%s'" % describe_token(token), token.lineno)
  File "c:\users\william\documents\microblog\venv\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: unexpected '/'
2018-05-31 09:49:00,738 INFO: Microblog startup [in C:\Users\William\Documents\microblog\app\__init__.py:49
